webpack.web.js

```diff
@@ -0,0 +1,44 @@
+// for web hosting viz. gh-pages
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
+const path = require('path');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = merge(common, {
+  entry: {
+    injector: {
+      import: path.resolve(__dirname, 'src', 'inject.js'),
+      dependOn: 'index'
+    }
+  },
+  mode: 'production',
+  optimization: {
+    minimize: true,
+    minimizer: [
+      new CssMinimizerPlugin({
+        minify: CssMinimizerPlugin.cleanCssMinify
+      }),
+      new TerserPlugin({
+        terserOptions: {
+          format: {
+            comments: false,
+          },
+        },
+        extractComments: false,
+      })
+    ]
+  },
+  module: {
+    rules: [{
+      test: /\.css$/i,
+      use: [MiniCssExtractPlugin.loader, 'css-loader'],
+    }],
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: '[name].[contenthash].css'
+    })
+  ]
+});
```

Comment:
Why did you create a whole new webpack config thats exactly the same as https://github.com/zombieFox/nightTab/blob/main/webpack.prod.js minus the zip plugin?

Would be helpful if you gave us more context in the body of your PR.

Reply:
True. I was just experimenting with the build scripts and didn't really need to build the zip file.
Also that script's useful when building the zip is not really necessary, viz. gh-pages or load the unpacked folder for testing.

Need to refactor this. Importing this from the prod script should be ok I think.

Reply:
Check #339 317118e5e49be27f09e28fdcd67f6f62c9b89636